ptf

```diff
@@ -50,61 +50,65 @@ DEBUG_LEVELS = {
 # The default configuration dictionary for PTF
 config_default = {
     # Miscellaneous options
-    "list"               : False,
-    "list_test_names"    : False,
-    "allow_user"         : False,
+    "list"                      : False,
```

Comment:
I am surprised by this change, I thought we were using a Python code formatter in CI now?

Reply:
Only for the `src/` folder, `ptf` is at the root. 
https://github.com/p4lang/ptf/blob/master/Makefile

Reply:
Could the ptf file be added to the Makefile targets in a follow-up PR?

Reply:
I create a separate PR to make this PR more clear. 
https://github.com/p4lang/ptf/pull/155

Reply:
Thanks, I have rebased this PR to include changes from #155.